### ISSUE TITLE
Cannot Use json key in $docker_pass in circleci orb

### ISSUE THREAD

**Issue by PavloRPsiuk** [anchore/ci-tools#5]
I'm trying to login using json key, but getting error:
```#!/bin/bash -eo pipefail
anchore-cli registry add eu.gcr.io $DOCKER_USER $DOCKER_PASS --skip-validate
Usage: anchore-cli registry add 
           [OPTIONS] REGISTRY REGISTRY_USER
           REGISTRY_PASS
Try "anchore-cli registry add --help" for help.

Error: Got unexpected extra arguments ("type": "service_account", "project_id": "my_project", "private_key_id": "{my_json_key}")
Exited with code 2
```

Circleci pipeline:
```...
workflows:
  scan_image:
    jobs:
      - anchore/image_scan:
          image_name: eu.gcr.io/my_project/my_image:tag
          private_registry: true
          registry_name: eu.gcr.io
          registry_user: _json_key
          registry_pass: $_gcr_json_key
...
```

How can I login with json key to a private registry?

**Comment by zhill** [comment]
The json token issues are usually around getting the shell to pass all the characters properly. See: https://anchore.freshdesk.com/support/solutions/articles/36000047968-working-with-google-container-registry-gcr-credentials for more detail on that for the CLI.

**Comment by paulros** [comment]
Could you please confirm whether its possible to use json key as $DOCKER_PASS in the orb. 
..because I'm getting error

**Comment by Btodhunter** [comment]
@paulros You are right, this is an issue with the ORB. Since your json key is secret it has to live in an ENV var. Due to in the key, and the way BASH handles variable expansion, the variable itself needs to be quoted when it's passed to a command.  In this case, the registry_pass parameter should be quoted when it's passed to the command within the ORB. I will be sure to update that ASAP.

**Comment by Btodhunter** [comment]
Fixed with 9d396cee547345a76ed0471ab99d691f91ef07c6

@paulros The ORB has been updated to allow GCR registries to be added. Use v1.2.1 for this functionality.
